i3lock.c

```diff
@@ -2246,18 +2103,200 @@ int main(int argc, char *argv[]) {
             case 905:
                 no_verify = true;
                 break;
+            case 906:
+                api_enabled = true;
+                break;
+            case 907:
+                api_fifo_path = optarg;
```

Comment:
Would `strdup` be needed here?

Reply:
Maybe, I didn't have problems without it, but it would be better to have it.